app/components/faq/SetupInstructions.component.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Grid, List, Header } from 'semantic-ui-react';
+import SetupInstructionsA from "./images/SetupInstructionsA.png";
+import SetupInstructionsB from "./images/SetupInstructionsB.png";
+import SetupInstructionsC from "./images/SetupInstructionsC.png";
+import SetupInstructionsD from "./images/SetupInstructionsD.png";
+import SetupInstructionsE from "./images/SetupInstructionsE.png";
+import './style.scss';
+
+const SetupInstructions = () => (
+  <React.Fragment>
+    <Header id="set-up-instructions" className="strava-hue" as='h1' textAlign='center'>Set up Instructions</Header>
+    <Grid className="step-text" stackable centered style={{ paddingTop: '2rem' }}>
+      <Grid.Column textAlign="left" verticalAlign='middle' width={6} style={{ paddingBottom: '3rem', paddingTop: '2rem' }}>
+        <List>
+          <List.Item>1. Connect your Fitbit or Strava account to Capco Challenge platform. Open the challenge website.</List.Item>
+          <List.Item>2. Enter your Capco 4-letter ID in the 'Register' section on this page for either Strava or Fitbit.</List.Item>
+        </List>
+      </Grid.Column>
+      <Grid.Column width={6}>
+        <img src={SetupInstructionsA}></img>
```

Comment:
This can be simplified as `<img src={SetupInstructionsA}/>`. 
Can you add alt text? It's a required attribute.